source/_integrations/siren.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: Siren
+description: Instructions on how to set up siren devices within Home Assistant.
+ha_category:
+  - Siren
+ha_release: '2021.5'
+ha_domain: siren
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+  - '@raman325'
+ha_iot_class:
+---
+
+The `siren` integration is built for the controlling and monitoring of siren/chime devices.
+
+## Services
+
+### siren services
+
+Available services: `siren.turn_on`, `siren.turn_off`, `siren.toggle`
+
+<div class='note'>
+
+Not all siren services may be available for your platform. Be sure to check the available services Home Assistant has enabled by checking the Services page in the [Developer Tools](/docs/tools/dev-tools/).
```

Comment:
```suggestion
Not all siren services may be available for your device. Be sure to check the available services Home Assistant has enabled by checking the Services page in the [Developer Tools](/docs/tools/dev-tools/).
```

Additionally, This line isn't correct I think, as it doesn't rely on integration implementation for these services to be available or not.

Reply:
All three services `turn_on`, `turn_off`, and `toggle` are dependent on a feature flag, so wouldn't that make it integration specific?

Reply:
Sure, however, it will still be `siren.turn_on`, `siren.turn_off` and `siren.toggle`. Even if you have a single integration that just supports one of those, all of them would be available (as the integration platform is loaded).

Reply:
I see your point. I got that from here: https://www.home-assistant.io/integrations/humidifier/

Perhaps a new PR is needed to remove it from there as well